assembl/static2/webpack.config.js

```diff
@@ -1,11 +1,41 @@
+/*
+Once you have made changes to this file, you have to run `supervisorctl restart dev:webpack` to see the effect.
+*/
+
 var path = require('path');
 var webpack = require('webpack');
+var ExtractTextPlugin = require("extract-text-webpack-plugin");
+var glob = require('glob');
+var _ = require('lodash');
+
+
+function theme_entries() {
+    var entries = {},
+        paths = glob.sync('./css/themes/**/*_web.scss'),
+        i, path, parts, name;
+    for (i = 0; i < paths.length; i++) {
+        path = paths[i];
+        parts = path.split('/');
+        name = 'theme_' + parts[parts.length - 2] + '_web';
+        entries[name] = path;
+    }
+    paths = glob.sync('./css/themes/**/*_notifications.scss');
+    for (i = 0; i < paths.length; i++) {
```

Comment:
Ah you're using lodash. Makes this easier to read with a `forEach`. I'm nitpicky 😛